frontend/README.md

```diff
@@ -17,3 +17,20 @@ You can build application for production using `npm run build` and then locally
 ## Health endpoint
 
 Endpoint available at `/healthz` path. Provides response if frontend is currently running.
+
+## Documentation page
+
+One of the frontend's features is displaying the documentation about document model.
+The documentation is provided to the service externally as raw typedoc output in markdown format.
+
+To display the documentation on the frontend the provided data has to be processed and represented in the form of a single file.
+This process goes as follows:
+
+1. Put the documentation into the `/docs` directory
+2. Run the script for generation of the single documentation file that frontend consumes.
+
+```bash
+$ npm run mkdocs
+```
+
+3. The documentation that is generated will end up at `./content/documentation/index.md` file.
```

Comment:
Since it's not the final process, I suggest to make it very short, describing current and intended flow with TODO

Regarding intended flow: I expect that we will need to get docs from the `../api/node_modules` and therefore dockerfile will need to be edited to include bigger context (probably can be done in the next PR)